examples/basic/logo_an.go

```diff
@@ -1,3 +1,5 @@
+//+build examples
```

Comment:
Where here, may I suggest renaming `logo_an.go` to `logo_anim.go`? This would make the name clearer, at least to me as I had no prior association to `an` as an abbreviation for animation.